docs/tempo/website/traceql/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: TraceQL
+menuTitle: TraceQL
+description: Learn about TraceQL, the query language for traces
+weight: 450
+aliases:
+  - /docs/tempo/latest/traceql/
+keywords:
+  - Tempo query language
+  - query language
+  - TraceQL
+---
+
+# TraceQL
+
+Inspired by PromQL and LogQL, TraceQL is a query language designed for selecting traces in Tempo. A TraceQL query can select traces based on:
+
+- Span and resource attributes, timing, and duration
+- Structural relationships between spans
+- Aggregated data from the spans in a trace
+
+The TraceQL language uses similar syntax and semantics as [PromQL](https://grafana.com/blog/2020/02/04/introduction-to-promql-the-prometheus-query-language/) and [LogQL](https://grafana.com/docs/loki/latest/logql/), where possible. TraceQL recognizes two types of data: intrinsics, which are fundamental to spans, and attributes, which are customizable key-value pairs.
+
+TraceQL requires Tempo’s Parquet columnar format to be enabled. For information on enabling Parquet, refer to the [Apache Parquet backend](https://grafana.com/docs/tempo/latest/configuration/parquet/) Tempo documentation.
+
+## How does it work?
+
+The TraceQL engine connects the Tempo API handler with the storage layer. The TraceQL engine:
```

Comment:
Personally this is too "deep dive" to me. I think this document should be impactful for immediately getting people starting writing TraceQL.

Reply:
I'm going to move this section to its own page